src/components/control_bar/_variables.scss

```
@@ -0,0 +1,4 @@
+$controlBarBackground: lightOrDarkTheme($euiColorDarkestShade, $euiColorInk);
+$controlBarText: lighten(makeHighContrastColor($euiColorLightestShade, $controlBarBackground), 20%);
+$controlBarBorderColor: rgba(255, 255, 255, .2);
```

You can use `transparentize` for this so you use an SASS var 
```suggestion
$controlBarBorderColor: transparentize($euiColorGhost, .8);
```